move/utils/sources/utils/utils.move

```diff
@@ -0,0 +1,19 @@
+/// Module: utils
+module utils::utils;
+
+use sui::bcs::{Self, BCS};
+
+/// Remaining data after BCS decoding
+const ERemainingData: u64 = 101;
```

Comment:
errors should always start from `0` within the module (i think move should show module location of the error in debugging info?)

Reply:
It will show the location of caller module instead of the location of macro function in this case. this is unlike normal functions; the macro functions will be executed the operation inline. this fact doubts me whether the error value cause any conflict with existing error code within the caller module, so i assigned the value that large enough to avoid conflict with the caller module.

Reply:
hmm, i see, let's use error strings instead as recommended by the mysten team. that addresses this issue as well